s3_to_socrata.py

```diff
@@ -139,24 +163,31 @@ def main(args):
     )
     socrata_client = get_socrata_client()
 
-    # Get a list of the files in the the S3 Bucekt
+    # Get a list of the files in the S3 Bucket
     file_list = aws_list_files(aws_s3_client)
```

Comment:
One thing I've learned is that this will cap out at 1,000 S3 objects, I think we're just adding one json file so shouldn't be a big deal but if we needed to add a ton more then this would break things I think.